sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DoNotHardcodeCredentials_DefaultValues.cs

```diff
@@ -27,7 +30,35 @@ public void Test()
             string myPassword1 = null;
```

Comment:
Please add theese tests to reflect new regex. Both lines behaves the same in production, but the second test will fail with current code.

            foo = "Server=localhost; Database=Main; User=Admin; Password=TopSecret;";       //Noncompliant
            foo = "Server=localhost; Database=Main; User=Admin; Password   =   TopSecret;"; //Noncompliant

Reply:
Ticket created to fix this: #2783